frontend/javascripts/oxalis/view/right-menu/meshes_view.js

```diff
@@ -171,6 +179,20 @@ class MeshesView extends React.Component<
       Store.dispatch(setPositionAction(seedPosition));
     };
 
+    const getComputeMeshFileButton = () => (
+      <Button
+        onClick={() => {
+          startComputeMeshFileJob(
```

Comment:
in addition to calling this function, a simple toast should also appear. maybe something like this:

```
Toast.success("The computation of a mesh file was started successfully. Depending on the datset's size this may take a bit. You don't need to keep this dataset open for the computation to continue.");
```